examples/config-api/config_api.out

```diff
@@ -20,23 +20,23 @@ $ ballerina run config_api.bal
 ballerina: enter secret for config value decryption:
 
 
-# To explicitly specify a configuration file, use either the `--config` or the `-c` flag.
-# If this flag is not set, Ballerina looks for a `ballerina.conf` file in the directory in which the source files
+# To explicitly specify a configuration file, use `--b7a.config=<path to configuration file>` property.
```

Comment:
Is it possible to change this property to `b7a.config.file` or something? The convention we have used for params is `<module-name>.<whatever-param-name>`.

Reply:
Initial i added like b7a.config.file.path and later change that to b7a.config since bit shorter.

Reply:
Lets change it to b7a.config.file as per convention.